packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/dashboard/manage-nsx-edges/dedicatedCloud-datacenter-manage-nsx.routing.js

```diff
@@ -3,11 +3,29 @@ export default /* @ngInject */ ($stateProvider) => {
     'app.dedicatedCloud.details.datacenter.details.dashboard.nsx',
     {
       resolve: {
-        goBack: /* @ngInject */ (goBackToDashboard) => goBackToDashboard,
+        goBack: /* @ngInject */ (trackClick, goBackToDashboard) => () => {
+          trackClick(
+            'dedicated::dedicatedCloud::details::datacenter::details::dashboard::nsx::cancel',
```

Comment:
```suggestion
            `${TRACKING_PREFIX}::cancel`,
```

Reply:
I can't use the constant in this file since constants.js and nsx.routing.js a re in complete different locations in manager (on in app/components the other in app/dedicatedCloud)